### DISCUSSION TITLE
Not able to trigger onDataLoad event for MVTLayer

### DISCUSSION THREAD

**Discussion by gireeshbhogireddy** [visgl/deck.gl#8361] [General]
I am unable to trigger onDataLoad property of MVTLayer when the data is fully loaded, codebase is in below URL:
https://codepen.io/gireesh-bhogireddy/pen/LYqoMoV?editors=0011
Please let me know what's going wrong here.

thanks

**Comment by Pessimistress** [comment]
Tiled layers never load the entirety of their data. You want to use [onViewportLoad](https://deck.gl/docs/api-reference/geo-layers/tile-layer#onviewportload).
